application.py

```diff
@@ -35,17 +35,21 @@ class QueryStringError(Exception):
 
 class Vertex:
 
-    def __init__(self, cellId, x, y):
+    def __init__(self, cellId, x, y, z=None):
         self.cellId = cellId
         self.x_coord = x
         self.y_coord = y
+        self.z_coord = z
 
     def x(self):
         return float(self.x_coord)
 
     def y(self):
         return float(self.y_coord)
 
+    def z(self):
+        return float(self.z_coord)
```

Comment:
is it ok that `z_coord` is None when this is called? (i.e., either you guarantee that you'll never call z() when `z_coord` is `None`, or the exception is handled)

Reply:
Yep, I'll handle it when we get a 3D visualization option